runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,68 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({opts})                                         *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range. This method does not make an LSP request, but just queries locally
+    stored inlay hints. The returned inlay hints are unresolved.
```

Comment:
Could we leave the resolved state undefined or mention that it might be unresolved?

It would keep the door open to do internal resolving when we expose other functionality without introducing a behaviour change

Reply:
Sure, left it open